shard.yml

```diff
@@ -50,3 +50,5 @@ development_dependencies:
     github: askn/faker
   webmock:
     github: manastech/webmock.cr
+  timecop:
+    github: crystal-community/timecop.cr
```

Comment:
Added Timecop to Mock Time for testing that the history gets added correctly as a booking progresses.